package.json

```diff
@@ -2,10 +2,10 @@
   "name": "kLiveController",
   "version": "1.0.0",
   "repository": {
-      "type": "git",
-      "url": "https://github.com/kaltura/liveDVR"
-    },
-  "license": "",
+    "type": "git",
+    "url": "git+https://github.com/kaltura/liveDVR.git"
+  },
+  "license": "ISC",
```

Comment:
I don't know. This property was added after automatic init to add grunt-execute.